__tests__/Model.test.js

```diff
@@ -1,24 +1,36 @@
 const Schema = require('../lib/Schema');
-const Model = require('../lib/Model');
+const { Model } = require('../lib/Model');
+const { deleteFile, readDirectoryJSON } = require('../lib/file-system.js');
+const fs = require('fs').promises;
 
-describe('Model class', () => {
-  it('creates a new document', () => {
-    const schema = new Schema({
-      name: {
-        type: String,
-        required: true
-      },
-      age: {
-        type: Number,
-        required: true
-      },
-      weight: {
-        type: String
-      }
-    });
+const schema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  age: {
+    type: Number,
+    required: true
+  },
+  weight: {
+    type: String
+  }
+});
 
-    const Dog = new Model('Dog', schema);
+const Dog = new Model('Dog', schema);
 
+describe('Model class', () => {
+  
+  afterEach(() => {
+    return readDirectoryJSON(Dog.modelName)
+      .then(dogs => dogs.forEach(dog => deleteFile(`${Dog.modelName}/${dog.id}`)));
+  });
+  
+  afterAll(() => {
+    return fs.rmdir(Dog.modelName);
+  });
+  
```

Comment:
Great job Man! Really like the way your whitespace is looking and how well organized it is! Keep it up!